build.ps1

```diff
@@ -262,6 +262,13 @@ function cleanExtension([string] $bitness) {
             Remove-Item -Recurse -Force $_.FullName
         }
     }
+
+    if ($bitness -eq "64bit") {
+        $keepRuntimes = @('win', 'win-x64', 'win10-x64')
+    } elseif ($bitness -eq "32bit") {
+        $keepRuntimes = @('win', 'win-x86', 'win10-x86')
+    }
+    Get-ChildItem "$privateSiteExtensionPath\$bitness\runtimes" -Exclude $keepRuntimes | Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
```

Comment:
> $privateSiteExtensionPath\$bitness\runtimes

Shouldn't it be `$privateSiteExtensionPath\$bitness\workers\powershell\runtimes`?

Reply:
You're right. It's hard for me to change this because I can't really test it. The host won't build on macOS. Trying out on a Windows machine.

```suggestion
    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\powershell\runtimes" -Exclude $keepRuntimes | Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
```

Reply:
We cannot try on mac anyways as we removed all the unix runtime folders and it won't work on mac.